test/parallel/test-http-set-max-idle-http-parser.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const httpCommon = require('_http_common');
+const http = require('http');
+
+assert.strictEqual(httpCommon.parsers.max !== 1, true);
+http.setMaxIdleHTTPParsers(1);
+assert.strictEqual(httpCommon.parsers.max, 1);
```

Comment:
Can we add tests that checks that passing non-integer values would throw?

```suggestion
[Symbol(), {}, [], () => {}, 1n, true, '1'].forEach((value) => {
  assert.throws(() => http.setMaxIdleHTTPParsers(value), { code: 'ERR_INVALID_ARG_TYPE' });
});
[-1, -Infinity, NaN, 0, 1.1].forEach((value) => {
  assert.throws(() => http.setMaxIdleHTTPParsers(value), { code: 'ERR_OUT_OF_RANGE' });
});

[1, Number.MAX_SAFE_INTEGER].forEach((value) => {
  assert.notStrictEqual(httpCommon.parsers.max, value);
  http.setMaxIdleHTTPParsers(value);
  assert.strictEqual(httpCommon.parsers.max, value);
});
```

Reply:
Thanks you very much. Done !